lib/helper/testcafe/testcafe-utils.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+
+const createTestFile = () => {
+  // TODO Generate unique test files (concurrent test runs)
+  const TESTFILE = path.join(__dirname, '_test.js');
```

Comment:
why do you generate it inside `lib/helper/testcafe`?
if you really need this file - put it into the `output` directory. 
Also, I think it's ok to use some sort UUID or time function to create a file if anyway you delete it in the end. This way you will be able to run parallel tests. 

But again is there a way to work without files?

Reply:
Will make them uuid and put them to output.

Would be glad if someone finds a way without generating test files, but I could not.